extensions/blocks/contact-form/editor.scss

```diff
@@ -81,6 +88,7 @@
 	margin: 0;
 	margin-bottom: 2px;
 	padding: 0;
+	width: auto;
```

Comment:
This _does_ make sense but in Chrome I didn't notice any difference. 🤔  🤷‍♂️

Reply:
Not all the time (not quite sure why to be honest...), but on small screens it was overflowing without this property.